web/src/js/ducks/modes/regular.ts

```diff
@@ -51,11 +52,13 @@ const regularReducer = (state = initialState, action): RegularState => {
             return {
                 ...state,
                 active: !state.active,
+                error: undefined,
             };
         case MODE_REGULAR_SET_PORT:
             return {
                 ...state,
                 listen_port: action.port as number,
+                error: undefined,
             };
         case UPDATE_OPTIONS:
         case RECEIVE_OPTIONS:
```

Comment:
We also need to potentially reset the error here, don't we?

Reply:
What do you mean? We're resetting it now. If there is an error from port X and we change the port, the error will reset. If everything is fine, no new error will be displayed. Otherwise, a new error will appear

Reply:
1. User enters an occupied port. -> error is set.
2. A different tab updates the currently active modes to just "regular" (8080 is not taken yet)
3. UPDATE_OPTIONS is triggered.
4. User will see updated config, but UI is still in the error state

I think we need to reset `error` as well if isActive is true.